src/components/ui/tabs.tsx

```
@@ -0,0 +1,55 @@
+"use client"
+
```

ESLint

[error] Missing semicolon.

(semi)